### PR TITLE
Added PageBlobClient.GetPageRangeDiff() previousSnapshotUrl header

### PR DESCRIPTION
**Pull request by seanmcc-msft**

### CODE REVIEW

File: sdk/storage/Azure.Storage.Common/tests/Shared/TestConfigurationsTemplate.xml [hunk 1]
```diff
@@ -104,5 +105,23 @@
       <FileServiceSecondaryEndpoint>http://[NAMESPACE-ACCOUNT]-secondary.file.core.windows.net</FileServiceSecondaryEndpoint>
       <TableServiceSecondaryEndpoint>http://[NAMESPACE-ACCOUNT]-secondary.table.core.windows.net</TableServiceSecondaryEndpoint>
     </TenantConfiguration>
+
+    <!-- A managed disk storge account -->
```
Comment (JoshLove-msft) [inline sdk/storage/Azure.Storage.Common/tests/Shared/TestConfigurationsTemplate.xml]: nit: typo

Also, how does this relate to the page range changes?

File: sdk/storage/Azure.Storage.Common/tests/Shared/TestConfigurations.cs [hunk 2]
```diff
@@ -110,6 +116,12 @@ public class TestConfigurations
         public static TenantConfiguration DefaultTargetHierarchicalNamespaceTenant =>
             GetTenant("TargetHierarchicalNamespaceTenant", s_configurations.Value.TargetHierarchicalNamespaceTenantName);
 
+        /// <summary>
+        /// Gets a tenant to use for any tests that require hierarchical namespace
```
Comment (JoshLove-msft) [inline sdk/storage/Azure.Storage.Common/tests/Shared/TestConfigurations.cs]: Summary needs to be updated.

File: sdk/storage/Azure.Storage.Common/tests/Shared/TestConfigurationsTemplate.xml (hunk 1 above)
Comment (seanmcc-msft) [reply sdk/storage/Azure.Storage.Common/tests/Shared/TestConfigurationsTemplate.xml]: Fixed.

The new header only works with a managed disk accounts.

Comment (JoshLove-msft) [reply sdk/storage/Azure.Storage.Common/tests/Shared/TestConfigurationsTemplate.xml]: Should we mention this in the doc comments for the parameter?

File: sdk/storage/Azure.Storage.Common/tests/Shared/TestConfigurations.cs (hunk 2 above)
Comment (seanmcc-msft) [reply sdk/storage/Azure.Storage.Common/tests/Shared/TestConfigurations.cs]: fixed.

**Review by amnguye — APPROVED** [review]

**Review by JoshLove-msft — APPROVED** [review]

**Pull request merged**